configuration/pih/liquibase/liquibase.xml

```diff
@@ -651,6 +651,7 @@
 				inner join patient_program pp on pp.patient_id  = o.person_id 
 					and pp.program_id = (select program_id from program where uuid = '41a2715e-8a14-11e8-9a94-a6cf71072f73') and pp.date_completed is null
 				where o.concept_id = concept_from_mapping('PIH',11665)
+				and o.obs_datetime >= pp.date_enrolled 
```

Comment:
so we do nothing if the obs datetime is before the date enrolled (as opposed to setting the state, but on the enrollment date instead of the obs date) (maybe this is a case that never really occurs...?)  @dmdesimone

Reply:
Well that SHOULD never happen because the same encounter that contains the "mothers group" obs should enroll them if they are unenrolled.  And in general, I just wanted to be a little more conservative and only update data in cases where we could be 100% sure what should be done.

Reply:
@mogoodrich ^^